account_invoice_merge_payment/README.rst

```diff
@@ -10,7 +12,28 @@ module to support fields added in Account Payment Partner from OCA/bank-payment
 Installation
 ============
 
-* The module is automatically installed when account_invoice_merge and account_payment_partner are installed
+* The module is automatically installed when account_invoice_merge and
+account_payment_partner are installed
+
+Usage
+=====
+
+Select several invoices from the same partner and click on **Merge Partner
+Invoice** in Action button menu.
```

Comment:
@cubells I've tested but I don't see this option, check:
![2017-03-07_17-57-29](https://cloud.githubusercontent.com/assets/8736623/23667912/b5edced8-035f-11e7-988c-894b6f060a4c.png)